CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -10,63 +10,156 @@
 
 class X2StrategyElement_DefaultActivities extends X2StrategyElement;
 
+
+var const string Guerilla;
+var const string Council;
+var const string SupplyRaid;
+var const string Radio;
+var const string Advent;
+var const string SupplyLift;
+var const string Resistance;
+var const string Rescue;
+var const string Ambush;
+var const string DarkEvent;
+var const string Facility;
+var const string Chosen;
+
 static function array<X2DataTemplate> CreateTemplates()
 {
 	local array<X2DataTemplate> Templates;
-	local string Guerilla, Council, SupplyRaid, SupplyLift, Radio, Advent, Resistance, Rescue, Ambush, DarkEvent/*, Chosen, Facility*/;
+	
+	CreateRecoverPersonnel(Templates, "RecoverPersonnel", "ResOps", default.Resistance);
```

Comment:
Since you have individual functions, there is no need to pass arguments into them (except the `Templates` array). Makes the functions harder to read and you don't actually reuse anything